src/icons/styles.js

```diff
@@ -0,0 +1,35 @@
+import React from 'react'
+import { resolve } from 'styled-jsx/css'
+import { colors } from '../theme.js'
+
+export const iconStyle = resolve`
```

Comment:
`resolve` is not the primary way to use `styled-jsx`, see comment above.

Reply:
see comment above, depends on what we agree on there